test/hotspot/jtreg/vmTestbase/nsk/jdi/ClassType/invokeMethod/invokemethod011t.java

```diff
@@ -91,7 +91,7 @@ static long dummyMeth(long l) throws InterruptedException {
         isInvoked = true;
         while(!doExit) {
             l--; l++;
-            Thread.currentThread().sleep(400);
+            Thread.currentThread().sleep(0);
```

Comment:
BTW this is an anti-pattern - use `Thread.sleep(n)` - it always applies to the current thread.

Reply:
What about replacing sleep(0) with yield()?
Do we expect it also causing deadlocks? Was it considered?

Reply:
sleep(0) on a virtual thread ends up doing a tryYield(), which is the same as what is done when calling sleep(0), so it should work, although I'm not so sure it is any better.